core/cron/crontabs.include.php

```diff
@@ -49,7 +52,27 @@
 include_once(SYS_PATH.'/core/cron/gym.cron.php');
 include_once(SYS_PATH.'/core/cron/pokemon.cron.php');
 include_once(SYS_PATH.'/core/cron/pokestop.cron.php');
-include_once(SYS_PATH.'/core/cron/nests.cron.php');
 if ($config->system->captcha_support) {
 	include_once(SYS_PATH.'/core/cron/captcha.cron.php');
 }
+
+// The following files are updated every 24h only because the queries are quite expensive
+// and they don't need a fast update interval
+$update_delay = 86400;
+$pokedex_rarity_file = SYS_PATH.'/core/json/pokedex.rarity.json';
+$nests_file = SYS_PATH.'/core/json/nests.stats.json';
+
+// Do not update both files at the same time to lower cpu load
```

Comment:
Basically means we run each updates one cron job apart from each other - this may still result in overlapping if one job takes longer than 15 min, where the next job is also launched!?
We should check if the previous update is finished, not if it was started.